commit 0da7e7c12da84b72fdde2d902750dba37824719a
Author: Isuru Fernando <isuruf@gmail.com>
Date:   Fri Mar 8 08:25:43 2019 -0600

    Workaround for non-working fltk-config
    
    fltk-config is a bash script and on windows, if there's no bash environment
    FLTK_VERSION will be empty and result in a CMake error. This fixes it
    by quoting the version

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c7c7f68f6..8db2fd251 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -544,7 +544,7 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/Fltk AND ENABLE_FLTK)
   if(FLTK_CONFIG_SCRIPT)
     execute_process(COMMAND ${FLTK_CONFIG_SCRIPT} --api-version
                     OUTPUT_VARIABLE FLTK_VERSION)
-    string(STRIP ${FLTK_VERSION} FLTK_VERSION)
+    string(STRIP "${FLTK_VERSION}" FLTK_VERSION)
     if(FLTK_VERSION GREATER 1.1)
       add_subdirectory(Fltk)
       set_config_option(HAVE_FLTK "Fltk")
